ckanext/ckanext-apicatalog_scheming/ckanext/apicatalog_scheming/schemas/dataset.json

```diff
@@ -144,52 +144,114 @@
       "preset": "apicatalog_organization_string_autocomplete"
     }
   ],
-  "resource_fields": [
-    {
-      "field_name": "url",
-      "label": "URL",
-      "preset": "resource_url_upload_ex",
-      "final": true
-    },
+"resource_fields": [
     {
       "field_name": "name",
-      "label": "Name",
-      "required": true,
-      "form_snippet": "text_ex.html",
-      "final": true
+      "label": "Title",
+      "additional_description": "Service name will be filled automatically and should not be edited",
+      "input_title": ["Service name"],
+      "only_default_lang_required": true,
+      "display_hr": true,
+      "final": true,
+      "form_placeholder": "ServiceName",
+      "form_languages": ["en"],
+      "form_snippet": "fluent_title.html",
+      "display_snippet": null
     },
     {
       "field_name": "description_translated",
-      "label": "Description",
-      "preset": "fluent_core_markdown_translated",
-      "form_placeholder": "eg. A detailed description",
-      "form_languages": ["fi", "sv", "en"]
+      "label": "Service description",
+      "extra_label": "A general, concise, and easy-to-understand description of the organization",
+      "input_title": ["Description in Finnish", "Description in Swedish", "Description in English"],
+      "validators": "fluent_text mark_as_modified_in_catalog_if_changed",
+      "preset": "fluent_core_title_translated",
+      "required": false,
+      "only_default_lang_required": true,
+      "form_placeholder": "Write subsystem description",
+      "form_languages": ["fi", "sv", "en"],
+      "form_snippet": "fluent_textarea.html",
+      "display_snippet": null,
+      "form_attrs": {}
+    },
+    {
+      "field_name": "access_restriction_level",
+      "label": "Service visibility",
+      "extra_label": "Service visibility",
+      "additional_description": "When service visibility is private, it is only visible to the owner organisation or users from allowed organisations",
+      "validators": "boolean_validator valid_resources",
+      "form_snippet": "radio.html",
+      "display_hr": false,
+      "choices": [
+        {
+          "value": "public",
+          "label": "Public",
+          "checked": true
+        },
+        {
+          "value": "private",
+          "label": "Private",
+          "checked": false
+        }
+      ]
+    },
+    {
+      "field_name": "allowed_organizations",
+      "label": "Allowed Organizations",
+      "classes": ["control-full"],
+      "preset": "apicatalog_organization_string_autocomplete",
+      "form_placeholder": "Allowed Organisations",
+      "validators": "ignore_missing"
     },
     {
       "field_name": "chargeability",
       "label": "Chargeability",
-      "preset": "select",
-      "classes": ["control-full"],
+      "extra_label": "Other information and settings",
+      "validators": "boolean_validator valid_resources",
+      "form_snippet": "radio.html",
```

Comment:
We want to keep this as a dropdown with empty field as an option, even though the design says otherwise.

Reply:
Updated